File: spring-cloud-starter/src/main/java/com/jmsoftware/maf/springcloudstarter/quartz/entity/GetQuartzJobConfigurationPageListItem.java
package com.jmsoftware.maf.springcloudstarter.quartz.entity;

import lombok.Data;

import java.time.LocalDateTime;

/**
 * <h1>GetQuartzJobConfigurationPageListItem</h1>
 * <p>
 * Change description here.
 *
 * @author Johnny Miller (鍾俊), email: johnnysviva@outlook.com, 9/27/21 12:07 AM
 **/
@Data
public class GetQuartzJobConfigurationPageListItem {
    private Long id;
    private String name;
    private String group;
    private String serviceName;
    private String invokeTarget;
    private String cronExpression;
    private Byte misfirePolicy;
    private Byte concurrent;
    private String description;
    private Byte status;
    private String createdBy;
    private LocalDateTime createdTime;
    private String modifiedBy;
    private LocalDateTime modifiedTime;
}

File: spring-cloud-starter/src/main/kotlin/com/jmsoftware/maf/springcloudstarter/quartz/entity/GetQuartzJobConfigurationPageListItem.kt
package com.jmsoftware.maf.springcloudstarter.quartz.entity

import java.time.LocalDateTime

/**
 * # GetQuartzJobConfigurationPageListItem
 *
 * Change description here.
 *
 * @author Johnny Miller (锺俊), e-mail: johnnysviva@outlook.com, date: 4/14/22 10:38 PM
 */
class GetQuartzJobConfigurationPageListItem {
    var id: Long? = null
    var name: String? = null
    var group: String? = null
    var serviceName: String? = null
    var invokeTarget: String? = null
    var cronExpression: String? = null
    var misfirePolicy: Byte? = null
    var concurrent: Byte? = null
    var description: String? = null
    var status: Byte? = null
    var createdBy: String? = null
    var createdTime: LocalDateTime? = null
    var modifiedBy: String? = null
    var modifiedTime: LocalDateTime? = null
}
